receiver/receiver.go

```diff
@@ -13,6 +13,12 @@ import (
 )
 
 var (
+	totalJobsUsage = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: fmt.Sprintf("ci_total_jobs_usage"),
+		Help: fmt.Sprintf("How many times any of the jobs were usage"),
```

Comment:
A better description would probably be: "How many times any of the jobs were usage"